packages/webapp/src/utils/user.tsx

```diff
@@ -1,6 +1,6 @@
 import storage, { LocalStorageKeys } from '../utils/local-storage';
 import { useLogoutAPI } from '../utils/api';
-import { useNavigate } from 'react-router';
+import { useNavigate } from 'react-router-dom';
```

Comment:
This is done in quite a few other places, importing from react-router vs. react-router-dom

Probably a search and replace would be a quick win for this
```
import { useNavigate } from 'react-router';
```

Reply:
Nice!